starter-code/server.js

```diff
@@ -134,7 +135,7 @@ app.delete('/articles', function(request, response) {
 });
 
 // COMMENT: What is this function invocation doing?
-// Put your response here...
+// RESPONSE: loading articles to database, check if table has already exist, send an error message
```

Comment:
So this creates the articles database if it does not already exist, and then loadArticles() is being called to load the raw Data.